### DISCUSSION TITLE
Aptos Connect sign message INVALID_SIGNATURE (sign the same message using Petra OK)

### DISCUSSION THREAD

**Discussion by nhandl3** [aptos-labs/aptos-developer-discussions#385] [General]
https://github.com/aptos-labs/aptos-connect-feedback/issues/1
I got this error. Appreciate any help. Thank you!

**Comment by darren-wangg** [comment]
You may have to logout and log back in on [Aptos Connect](https://aptosconnect.app). Or clear cache if the issue still persists.

This might be caused by an expired oauth token.

**Reply by nhandl3** [reply]
Logout and log back in on [Aptos Connect](https://aptosconnect.app/) solves the issue perfectly. I agree with you that an expired oauth token might be the root cause. Anyway thanks for your hint!
